scopes/component/isolator/isolator.main.runtime.ts

```diff
@@ -737,6 +747,27 @@ export class IsolatorMain {
     return allCapsuleList;
   }
 
+  private async calcDependenciesGraph(
+    capsuleList: CapsuleList,
+    components: Component[],
+    capsulesDir: string
+  ): Promise<void> {
+    const componentIdByPkgName = this.dependencyResolver.getComponentIdByPkgNameMap(components);
+    const opts = {
+      componentIdByPkgName,
+      capsulesDir,
+    };
+    await Promise.all(
+      capsuleList.map(async (capsule) => {
+        capsule.component.state._consumer.dependenciesGraph =
+          await this.dependencyResolver.calcDependenciesGraphFromCapsule(
+            path.relative(capsulesDir, capsule.path),
+            opts
+          );
```

Comment:
I would consider encapsulating this
so having a func on deps resolver something like `depsResolver.addDepsGraph(component, compRelativeDir, opts)`
this inside will do:
`capsule.component.state._consumer.dependenciesGraph = calcDependenciesGraphFromCapsule(...)`

Reply:
also do you really need the seperation of the regular calc vs capsule calc?
maybe you can just unify them and pass more options?
(if they are too different than maybe no, but if not, think about it)